packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -796,4 +808,36 @@
     <label>Voir moins</label>
     <name>quantic_GeneratedAnswerShowLess</name>
   </customLabels>
+  <customLabels>
+    <label>La réponse concerne-t-elle le bon sujet?</label>
+    <name>quantic_FeedbackQuestionTopic</name>
+  </customLabels>
+  <customLabels>
+    <label>La réponse est-elle exempte de contenu halluciné?</label>
```

Comment:
same comment here as above in the english file

Reply:
Same answer :)